Resources/Prototypes/_LostParadise/Entities/Mobs/Species/Tajaran/Body/Organs/Tajaran.yml

```
@@ -4,10 +4,17 @@
   components:
   - type: Stomach
-    specialDigestible:
-      tags:
-      - Meat
-      - Pill
+    poorlyDigestibleFood:
+      reducedReagents:
+        - Nutriment
+        - Vitamin
+      whitelistTags:
+        - Fruit # Bread/Pizza tags aren't included according to the LostParadise's lore
```

yamllint

[warning] too few spaces before comment

(comments)